src/App.js

```diff
@@ -1,10 +1,31 @@
-import './App.css';
-
+import { Fragment } from "react";
+import { GlobalStyle } from "./components/style";
+import Home from "./pages/home/Home";
+import {Route,Routes} from 'react-router-dom'
+import CategoryPage from "./pages/category/CategoryPage";
+import QuePage from "./pages/quePage/QuePage";
+import ResultPage from "./pages/result/ResultPage";
+import CategoryRules from "./components/Rules/CategoryRules";
+import Quiz from "./components/quizes/Quiz";
+import { lightTheme } from "./components/style/stylecomponents/themes";
+import { darkTheme } from "./components/style/stylecomponents/themes";
+import { useSelector } from "react-redux";
 function App() {
```

Comment:
always give one line space b/n import files and function

Reply:
ok sir🚀